app-modules/contact/src/Filament/Widgets/ContactStats.php

```diff
@@ -52,7 +52,7 @@ protected function getStats(): array
 
         return [
             Stat::make('Contacts', Number::abbreviate(
-                Cache::tags(['contacts'])
+                Cache::tags(['{{contacts}}'])
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.